packages/scroller/src/vaadin-scroller.js

```diff
@@ -130,13 +130,13 @@ class Scroller extends FocusMixin(ElementMixin(ControllerMixin(ThemableMixin(Pol
   /**
    * Scrolls the textarea to the end if it has a vertical or horizontal scrollbar.
    */
-  scrollToEnd(textDirection = 'ltr') {
+  scrollToEnd(direction = 'ltr') {
     switch (this.scrollDirection) {
       case 'vertical':
         this.scrollTop = this.scrollHeight;
         break;
       case 'horizontal':
-        this.scrollLeft = textDirection === 'ltr' ? this.scrollWidth : -this.scrollWidth;
+        this.scrollLeft = direction === 'ltr' ? this.scrollWidth : -this.scrollWidth;
```

Comment:
It's better to detect direction automatically by using this internal getter:

https://github.com/vaadin/web-components/blob/d3dad1454aea08d5e4197bcb2a56e8cfb319b264/packages/component-base/src/dir-mixin.js#L66-L68

```suggestion
        this.scrollLeft = this.__isRTL ? -this.scrollWidth : this.scrollWidth;
```